packages/bun-types/bun.d.ts

```diff
@@ -156,6 +225,16 @@ declare module "bun" {
      * ```
      */
     blob(): Promise<Blob>;
+
+    /**
+     * Configure the shell to not throw an exception on non-zero exit codes.
+     */
+    nothrow(): this;
```

Comment:
Should this be `noThrow` for consistent camel case (like`exitCode`)?

Reply:
`nothrow()` is from zx's API, but I guess it makes more sense to make it camel case

Reply:
So `.throws()` would be treated the same as `.noThrow()` (by acting like `.throws(false)`)?

Reply:
> So `.throws()` would be treated the same as `.noThrow()` (by acting like `.throws(false)`)?

I was talking about `nothrow` vs `noThrow`.

Reply:
im not sure how i feel about having this and `throws()`, and then desire to add a third function.

i think we can just say in the doc comment this is equivilent to `throws(false)` and mention the zx api.